deptofdefense/move.mil
# Add installations model

## Checklist

I have…


## Summary of Changes

This pull request adds an installation model, migration, and seed data. This model will be used _at least_ by the next iteration of Locator Maps and by the PPM Estimator.

Test coverage takes a small hit since the model isn't yet in use. Follow-on PRs that utilize the model will improve test coverage.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-installations`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `bin/rails rake`,
1. run `bin/rails brakeman`,
1. run `bin/rails rubocop`,
1. run `bin/rails server`,
1. load up http://localhost:3000 in your Web browser of choice and observe that everything works as it always has!
